packages/demo/usecase-demo/src/modules/createNew/CreateNewContent.tsx

```diff
@@ -275,10 +275,6 @@ export function CreateNewContent() {
               {fileName && (
                 <>
                   <div className="mt-8 flex w-11/12 items-center justify-center gap-x-1.5">
-                    <CheckCircle
-                      size="20"
-                      className="text-success-foreground"
-                    />
```

Comment:
Removed green "Check" icon that can be misleading: if selected file is too big, there will be an error.